src/Microsoft.DotNet.Build.Tasks/PackageFiles/PublishProduct.targets

```diff
@@ -31,14 +31,20 @@
     <ItemGroup>
       <NuGetPushCommand Include="@(PackagesToPush -> '$(NuGetExePath) $(NuGetPushArgsBase) %(Identity)')" />
 
-      <!-- There's a very special failure scenario that we want to ignore.  That scenario is
-           when NuGet hits a timeout on one "push" attempt, and then gets a "Forbidden" response
-           because the package "already exists" on the next response.  This indicates that the
-           timeout occurred, but the push was actually successful.
+      <!-- There are special failure scenarios that we want to ignore.  Those scenarios are
+           when NuGet reports a failure during a push attempt for something server or timeout related, 
+           and then gets a "Forbidden" response on the next attempt because the package "already exists".  
+           This indicates that the failure was reported, but the push was actually successful.
       -->
       <IgnorableErrorMessages Include="Overwriting existing packages is forbidden according to the package retention settings for this feed.">
         <ConditionalErrorMessage>Pushing took too long</ConditionalErrorMessage>
       </IgnorableErrorMessages>
+      <IgnorableErrorMessages Include="Overwriting existing packages is forbidden according to the package retention settings for this feed.">
```

Comment:
I'm leaning towards putting this in the propertygroup. I doubt it'll change, but IMO it would make it slightly easier to know they're exactly the same when reading.